backend/coins/btc/handlers/handlers.go

```diff
@@ -381,15 +391,37 @@ func (handlers *Handlers) getUTXOs(*http.Request) (interface{}, error) {
 }
 
 func (handlers *Handlers) getAccountBalance(*http.Request) (interface{}, error) {
-	balance, err := handlers.account.Balance()
+	type balance struct {
+		HasAvailable bool            `json:"hasAvailable"`
+		Available    FormattedAmount `json:"available"`
+		HasIncoming  bool            `json:"hasIncoming"`
+		Incoming     FormattedAmount `json:"incoming"`
+	}
+
+	type result struct {
+		Success   bool    `json:"success"`
+		ErrorCode string  `json:"errorCode,omitempty"`
+		Balance   balance `json:"balance,omitempty"`
+	}
```

Comment:
This does not match the frontend type:

```
export type BalanceResponse = {
 success: true;
  balance: IBalance;
} | {
  success: false;
  errorMessage: string
}
```

Reply:
Fixed and removed the errocode in the api as discussed